sdk/monitor/azure-monitor-opentelemetry-exporter/samples/authentication/sample_managed_credential.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+"""
+An example to show an application using Opentelemetry tracing api and sdk. Custom dependencies are
```

Comment:
Change these comments to accurately reflect what is being sampled.

Reply:
Done.